docs/tutorials/sampler.ipynb

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
<p>We don't have error mitigation yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>